package.json

```diff
@@ -3,27 +3,29 @@
   "version": "1.3.0",
   "description": "HPCC Visualization Framework",
   "devDependencies": {
-    "fs": "^0.0.2",
     "async": "^0.9.0",
+    "csso": "^1.3.11",
+    "del": "^1.1.1",
+    "fs": "^0.0.2",
     "gulp": "^3.8.11",
+    "gulp-bump": "^0.3.0",
     "gulp-concat-css": "^2.2.0",
+    "gulp-filter": "^2.0.2",
+    "gulp-git": "^1.2.3",
+    "gulp-jscs": "~1.6.0",
+    "gulp-jshint": "~1.10.0",
     "gulp-minify-css": "^1.1.1",
     "gulp-natural-sort": "^0.1.0",
     "gulp-requirejs": "^0.1.3",
+    "gulp-tag-version": "^1.2.1",
     "gulp-util": "^3.0.4",
+    "jshint-stylish": "~1.0.2",
     "lodash": "^3.8.0",
     "node-dir": "^0.1.8",
+    "require-css": "^0.1.8",
     "requirejs": "^2.1.17",
-    "del": "^1.1.1",
-    "csso": "^1.3.11",
-    "gulp-git": "^1.2.3",
-    "gulp-bump": "^0.3.0",
-    "gulp-filter": "^2.0.2",
-    "gulp-tag-version": "^1.2.1",
-    "yargs": "^3.8.0",
-    "jshint-stylish": "~1.0.2",
-    "gulp-jshint": "~1.10.0",
-    "gulp-jscs": "~1.6.0"
+    "requirejs-plugins": "^1.0.2",
```

Comment:
This needs to use my branch, otherwise the visualizations will fail to load in DOJO AMD environments.